frontend/js/src/common/brainzplayer/BrainzPlayerUI.tsx

```diff
@@ -264,20 +271,29 @@ function BrainzPlayerUI(props: React.PropsWithChildren<BrainzPlayerUIProps>) {
         </div>
         <div className="actions">
           {isPlayingATrack && currentDataSourceName && (
-            <a
-              href={trackUrl || "#"}
-              className="music-service-icon"
-              aria-label={`Open in ${currentDataSourceName}`}
-              title={`Open in ${currentDataSourceName}`}
-              target="_blank"
-              rel="noopener noreferrer"
-            >
+            <>
+              <a
+                href={trackUrl || "#"}
+                className="music-service-icon"
+                aria-label={`Open in ${currentDataSourceName}`}
+                title={`Open in ${currentDataSourceName}`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FontAwesomeIcon
+                  icon={currentDataSourceIcon!}
+                  color={currentDataSourceIconColor}
+                />
+              </a>
               <FontAwesomeIcon
-                icon={currentDataSourceIcon!}
-                color={currentDataSourceIconColor}
+                icon={faVolumeUp}
+                style={{ color: showVolume ? "green" : "" }}
+                onClick={() => setShowVolume(!showVolume)}
+                className="hidden-sm hidden-xs"
```

Comment:
After seeing it in action, I think it should only be
```suggestion
                className="hidden-xs"
```